src/askui/models/router.py

```diff
@@ -238,8 +238,8 @@ def act(self, goal: str, model: str | None = None) -> None:
                 f"Routing act prediction to {ModelName.ANTHROPIC__CLAUDE__3_5__SONNET__20241022}"  # noqa: E501
             )
             return self._claude_computer_agent.act(goal)
-        if not model:
-            logger.debug("Routing act prediction to AskUiClaudeComputerAgent (default)")
+        if model == ModelName.ASKUI or not model:
```

Comment:
```suggestion
        if model == ModelName.ASKUI or model is None:
```

because users may also pass an empty string which should be treated differently